x-pack/plugins/observability_solution/apm/public/components/app/error_group_overview/constants.ts

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+export const logsOnlyEmptyStateContent: { title: string; content: string; imgName: string } = {
```

Comment:
WDYT of defining all empty states in the same folder where the `ServiceTabEmptyState` is? You can then define a common interface that all pages will follow:
```
interface EmptyStateContent {
  title: string;
  content: string;
   imgName: string;
}
```

Then you could either export each object definition like you're doing today and import them on each page and pass it down to the `ServiceTabEmptyState` as you're doing today.

---

Or, you could group the empty states in a single object, like:
```
type EmptyStateKey = 'errorGroupOverview' | 'infraOverview' | ...

const emptyStateDefinitions: Record<EmptyStateKey, EmptyStateContent> = {
 errorGroupOverview: {
   title: i18n.translate('xpack.apm.serviceTabEmptyState.errorGroupOverviewTitle', {
    defaultMessage: 'Identify transaction errors with your applications',
  }),
  content: i18n.translate('xpack.apm.serviceTabEmptyState.errorGroupOverviewContent', {
    defaultMessage:
      'Analyse errors down to the specific transaction to pin-point specific errors within your service.',
  }),
  imgName: 'service_tab_empty_state_errors.png',
  },
 infraOverview: {...},
 ...
}
```

Then you can change the `EmptyStateContent` component props to just accept a id property typed as `EmptyStateKey`.

```
 <EmptyStateKey id="errorGroupOverview" />
```

WDYT?

Reply:
This is similar to how it was done before implementing all the requested changes. Reverting to that approach, thank you for the suggestion 👍🏻